README.md

```diff
@@ -142,6 +142,7 @@ Thanks to the following for helping make this library better, whether through co
 - @Sytten
 - @thewh1teagle
 - @tmpfs
+- @unkcpz
```

Comment:
in the last commit, I add my name. It takes couple hours to figure out how to make this work ;) Fine if it is removed if such contribution is regard as too small.

Reply:
That's great you added your name. Thanks!